snakemake/wrapper.py

```diff
@@ -87,6 +88,40 @@ def get_conda_env(path, prefix=None):
     return path + "/environment.yaml"
 
 
+def get_container(path, prefix=None):
+    """Given a path to a rule, load the meta.yaml / meta.yml and determine if
+       a container is provided. If so, return the container URI. Otherwise
```

Comment:
I started this function mimicking the one to get the conda environment, but it's fairly messy! @johanneskoester let's talk about how to simplify this. We basically need to get a path to the meta.yaml, and read it, and return the container if it's defined.